include/rppt_tensor_audio_augmentations.h

```diff
@@ -93,6 +93,26 @@ RppStatus rppt_to_decibels_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_
  */
 RppStatus rppt_pre_emphasis_filter_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32s *srcLengthTensor, Rpp32f *coeffTensor, RpptAudioBorderType borderType, rppHandle_t rppHandle);
 
+/******************** down_mixing ********************/
+
+/*! \brief Down Mixing augmentation HOST
+* \details Down Mixing augmentation for audio data
+* \param[in] srcPtr source tensor memory
```

Comment:
Modify to match new style where we specify memory type "srcPtr source tensor in HOST memory" like other funcs